browser/brave_browser_features.cc

```diff
@@ -29,6 +29,12 @@ BASE_FEATURE(kBraveCopyCleanLinkByDefault,
 BASE_FEATURE(kBraveCopyCleanLinkFromJs,
              base::FEATURE_DISABLED_BY_DEFAULT);
 
+// Enables rounded corners on the main content areas by default (when the user
+// has not explicitly set the pref).
+BASE_FEATURE(kBraveRoundedCornersByDefault,
+             "brave-web-view-rounded-corners",
```

Comment:
We are using [an existing study](https://github.com/brave/brave-variations/blob/main/studies/BraveWebViewRoundedCornersStudy.json5) for this flag.